.github/workflows/team_label.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  pull_request_target:
```

Comment:
This runs under the context of the main branch, not the fork.
see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target